src/formatter-settings/assets/features/formatterSettings/FormatterSettingView.tsx

```diff
@@ -77,8 +73,8 @@ const FormatterSettingsView = (): JSX.Element => {
       <Row className="flex-grow-1 d-flex flex-nowrap view-body">
         <Col className="flex-grow-0">{naviBar}</Col>
         <Col className="d-flex view-content">
-          <Row className="flex-nowrap flex-column flex-lg-row d-flex w-100 h-100">
-            <Col className="flex-grow-0 setting-container d-flex flex-row flex-lg-column flex-lg-nowrap">{<Setting/>}</Col>
+          <Row className="flex-grow-1 flex-nowrap flex-column flex-lg-row d-flex w-100 h-100">
```

Comment:
to make the distance between the preview and the right border the same as that between the category and the left border.